compiler_opt/tools/sparse_bucket_generator.py

```diff
@@ -123,17 +135,23 @@ def _generate_vocab(feature_values_arrays, feature_name):
 
 
 def main(_) -> None:
+  gin.parse_config_files_and_bindings(
+      FLAGS.gin_files, bindings=FLAGS.gin_bindings, skip_unknown=False)
+  logging.info(gin.config_str())
+
   """Generate num_buckets quantiles for each feature."""
   tf.io.gfile.makedirs(FLAGS.output_dir)
   dataset = tf.data.Dataset.list_files(FLAGS.input)
   dataset = tf.data.TFRecordDataset(dataset)
+  problem_config = registry.get_configuration()
```

Comment:
nit: move it right after line 140 cause logically they are closer --- we parse the gin in order to get features_to_not_process